source/documentation/modules/math.html.md.erb

```diff
@@ -0,0 +1,494 @@
+    @debug math.$e; // 2.7182818285
+    ===
+    @debug math.$e // 2.7182818285
+  <% end %>
+<% end %>
+
+
+<% function 'math.$pi' do %>
+  Equal to the value of the mathematical constant `pi`.
+
+  <% example(autogen_css: false) do %>
+    @debug math.$pi; // 3.1415926536
+    ===
+    @debug math.$pi // 3.1415926536
+  <% end %>
+<% end %>
+
+
+## Bounding Functions
+
+<% function 'math.ceil($number)', 'ceil($number)', returns: 'number' do %>
+  Rounds `$number` up to the next highest whole number.
+
+  <% example(autogen_css: false) do %>
+    @debug math.ceil(4); // 4
+    @debug math.ceil(4.2); // 5
+    @debug math.ceil(4.9); // 5
+    ===
+    @debug math.ceil(4)  // 4
+    @debug math.ceil(4.2)  // 5
+    @debug math.ceil(4.9)  // 5
+  <% end %>
+<% end %>
+
+
+<% function 'math.clamp($min, $number, $max)', returns: 'number' do %>
+  Clamps `$number` in between `$min` and `$max`.
+
+  `$min`, `$number`, and `$max` must have compatible units, or all be unitless.
+
+  <% example(autogen_css: false) do %>
+    @debug math.clamp(-1, 0, 1); // 0
+    @debug math.clamp(0.5px, 10px, 5px); // 5px
+    @debug math.clamp(1cm, 10mm, 1in); // 1cm
+    ===
+    @debug math.clamp(-1, 0, 1) // 0
+    @debug math.clamp(0.5px, 10px, 5px) // 5px
+    @debug math.clamp(1cm, 10mm, 1in) // 1cm
+  <% end %>
+<% end %>
+
+
+<% function 'math.floor($number)', 'floor($number)', returns: 'number' do %>
+  Rounds `$number` down to the next lowest whole number.
+
+  <% example(autogen_css: false) do %>
+    @debug math.floor(4); // 4
+    @debug math.floor(4.2); // 4
+    @debug math.floor(4.9); // 4
+    ===
+    @debug math.floor(4)  // 4
+    @debug math.floor(4.2)  // 4
+    @debug math.floor(4.9)  // 4
+  <% end %>
+<% end %>
+
+
+<% function 'math.max($number...)', 'max($number...)', returns: 'number' do %>
+  Returns the highest of one or more numbers. A list of numbers can be passed
+  [using `...`][].
+
+  [using `...`]: ../at-rules/function#passing-arbitrary-arguments
+
+  <% example(autogen_css: false) do %>
+    @debug math.max(1px, 4px); // 4px
+
+    $widths: 50px, 30px, 100px;
+    @debug math.max($widths...); // 100px
+    ===
+    @debug math.max(1px, 4px)  // 4px
+
+    $widths: 50px, 30px, 100px
+    @debug math.max($widths...)  // 100px
+  <% end %>
+<% end %>
+
+
+<% function 'math.min($number...)', 'min($number...)', returns: 'number' do %>
+  Returns the lowest of one or more numbers. A list of numbers can be passed
+  [using `...`][].
+
+  [using `...`]: ../at-rules/function#passing-arbitrary-arguments
+
+  <% example(autogen_css: false) do %>
+    @debug math.min(1px, 4px); // 1px
+
+    $widths: 50px, 30px, 100px;
+    @debug math.min($widths...); // 30px
+    ===
+    @debug math.min(1px, 4px)  // 1px
+
+    $widths: 50px, 30px, 100px
+    @debug math.min($widths...)  // 30px
+  <% end %>
+<% end %>
+
+
+<% function 'math.round($number)',
+            'round($number)',
+            returns: 'number' do %>
```

Comment:
Nit: This signature only needs to be line-wrapped if it's particularly long, which it doesn't look like it is.

Reply:
Done.